recipes/parallel-preprocessor/conda_build_config.yaml

```diff
@@ -0,0 +1,7 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+    - '10.15'               # [osx]
```

Comment:
Considering the later comment, why not 10.13?

(I guess this is why you're skipping macOS for now as staged-recipes doesn't support changing macOS deployment target?)

Reply:
I am not a macos user,  and have no idea of 10.13 C++17 capacity,. 10.12 seems does not work. while mac is not a primary support platform.  so just make one version available,  hope in the future community may help and test on macos

Reply:
I installl macos10.15 in docker, to test build,  it is working/built with homebrew,  but I can not make it work in conda-build env. no idea, limited by resource, just disable macos.           parallel-preprocessor is just at its preliminary stage, support all platform is not a priority.